script/install

```diff
@@ -20,11 +20,8 @@ warn() {
   printf "${tty_red}Warning${tty_reset}: %s\n" "$(chomp "$1")"
 }
 
-# The line below extracts the version from the Constants.swift file by parsing the line:
-# public static let version = "1.48.1"
-# Note: We can't hit the API from this script because it has limits that might cause the installation to fail.
-LATEST_VERSION=$(curl --silent "https://raw.githubusercontent.com/tuist/tuist/main/Sources/TuistSupport/Constants.swift" | grep 'version =' | sed -E 's/.*"([^"]+)".*/\1/')
-
+# The line below extracts the version from the list of tags in the Tuist repository.
+LATEST_VERSION=$(git ls-remote -t --sort=v:refname https://github.com/tuist/tuist.git | sed -ne '$s/.*tags\/\(.*\)/\1/p')
```

Comment:
I've been wondering, is there a reason we're manually parsing the version number? GitHub allows for downloading the latest version using a static URL. Would it work to just use that in the curl command on line 28?

This URL downloads the latest version of the `tuistenv.zip` from releases: https://github.com/tuist/tuist/releases/latest/download/tuistenv.zip

Reply:
As I understand it, the GitHub latest release is determined by timestamp. So if 3.0.0 were released and then sometime later 2.9.1 were released, that link would point to 2.9.1.

Reply:
I created a dummy repo here to test: https://github.com/hisaac/test-latest-release/releases

It looks like as long as the tags follow standard semver format, they get organized by version number, not by creation time. I created the `3.0.0` release/tag first, and the `2.0.0` second, and it correctly put the `2.0.0` release below the `3.0.0` one. And using the "latest" download link, it downloads the `3.0.0` release's binary.

https://github.com/hisaac/test-latest-release/releases/latest/download/release.zip

Reply:
Nice! I think this simplifies a lot of stuff 🚀

Reply:
Here's an example of a repo where the latest is not the one with the highest version number: https://github.com/vaadin/vaadin-text-field/releases

Reply:
Ah bummer, it does have the "Latest" mapped to a lower number then. I wonder how/why it works that way here, but didn't work that way in my test repo. Anyway, carry on!